src/20_Components/amp-facebook.html

```diff
@@ -11,35 +11,37 @@
   <title>amp-facebook</title>
     <script async src="https://cdn.ampproject.org/v0.js"></script>
     <!-- ## Setup -->
-    <!-- Import the amp-facebook component -->
+    <!-- Import the `amp-facebook` component -->
     <script async custom-element="amp-facebook" src="https://cdn.ampproject.org/v0/amp-facebook-0.1.js"></script>
   <link rel="canonical" href="<%host%>/components/amp-facebook/">
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
   </head>
   <body>
 
-    <!-- ## Basic Usage -->
+    <!-- ## Basic usage -->
     <!-- The `amp-facebook` component only requires the URL of the facebook post... -->
     <amp-facebook width="552" height="303"
                     layout="responsive"
                     data-href="https://www.facebook.com/zuck/posts/10102593740125791">
     </amp-facebook>
 
-    <!-- ... or video -->
+    <!-- ## Video -->
+    <!--  -->
```

Comment:
Can you add a sentence here for consistency, e.g.: "Here is a sample for a facebook video embed."

Reply:
done